courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -843,6 +847,28 @@ Floating Point Type Attributes
    - Advanced machine representation of the floating-point type
    - Mantissa, strict mode
 
+---------------------------
+Numeric Types Conversion
+---------------------------
+
+* Ada's integer and real are :dfn:`numeric`
+
+    - Holding a numeric value
+
+* Special rule: they can always be converted inbetween them
```

Comment:
"in-between" or "in between"
Or maybe better: "Can always convert between integer and real types"